core/src/mindustry/world/blocks/experimental/BlockLoader.java

```diff
@@ -99,7 +100,7 @@ public void updateTile(){
 
                 //load up items
                 if(payload.block().hasItems && items.any()){
-                    if(timer(timerLoad, loadTime)){
+                    if(consValid() && efficiency() > 0.01f && timer(timerLoad, loadTime / efficiency())){
```

Comment:
consValid() check seems redundant when you're already checking for efficiency.

Reply:
enabled check, or is the enavled also already counted in the efficiency()?

Reply:
it works